src/monodroid/monodroid.targets

```diff
@@ -22,22 +22,41 @@
         Overwrite="True"
     />
     <Exec
-        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=$(Configuration) V=1"
+        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=$(Configuration) NDK_LIBS_OUT=./debug/libs NDK_OUT=./debug/obj V=1"
```

Comment:
Brilliant! Where'd you find `$(NDK_LIBS_OUT)` and `$(NDK_OUT)`?

That said, could we "reverse" the order from `debug\...` to `obj\debug\...`, placing the build artifacts within the `obj` directory? `obj` is already ignored by `git status`, while `debug` and `release` won't be, so by using `obj\{debug,release}` we'll reduce `git status` spew.